cmd/gemini/root.go

```diff
@@ -371,7 +371,7 @@ func createClusters(
 		return testCluster, nil
 	}
 	oracleCluster := gocql.NewCluster(oracleClusterHost...)
-	oracleCluster.Timeout = 120 * time.Second
+	oracleCluster.Timeout = requestTimeout
```

Comment:
What about the `connectTimeout`? It is also not set here.

Reply:
Added